### PR TITLE
explicitly install/upgrade centos-release-scl-rh during upgrade

### PR DESCRIPTION
**Pull request by evgeni**
also clean only yum metadata cache, this should be sufficient

### CODE REVIEW

**Review by tbrisker — APPROVED** [review]
Thanks @evgeni

File: _includes/manuals/1.20/3.6_upgrade.md [hunk 1]
```diff
@@ -38,17 +38,21 @@ appropriate foreman-release package for your operating system:
...
+Clean up the yum metadata cache:
 
-    yum clean all
+    yum clean metadata
 
-First install or upgrade the SCL release package to add the rh-ruby25 repository:
+First install or upgrade the SCL release packages to add the rh-ruby25, tfm and tfm-ror52 repositories:
 
-    yum install foreman-release-scl
+    yum install centos-release-scl-rh foreman-release-scl
```
Comment (ekohl) [inline _includes/manuals/1.20/3.6_upgrade.md:47]: `foreman-release-scl` depends on `centos-release-scl-rh` so please remove it from the instructions here.

File: _includes/manuals/1.20/3.6_upgrade.md [hunk 2]
```diff
@@ -38,17 +38,21 @@ appropriate foreman-release package for your operating system:
...
 
 or:
 
-    yum upgrade foreman-release-scl
+    yum upgrade centos-release-scl-rh foreman-release-scl
```
Comment (ekohl) [inline _includes/manuals/1.20/3.6_upgrade.md:51]: I'm not sure there's so much need to explicitly upgrade `centos-release-scl-rh`. If we need a certain version then `foreman-release-scl` should depend on it.

File: _includes/manuals/1.20/3.6_upgrade.md (hunk 1 above)
Comment (evgeni) [reply _includes/manuals/1.20/3.6_upgrade.md:47]: The idea here was to bring the user attention to the fact that we actually need more than *our* SCL. How would you rephrase that in text to drop the package name here?

File: _includes/manuals/1.20/3.6_upgrade.md (hunk 2 above)
Comment (evgeni) [reply _includes/manuals/1.20/3.6_upgrade.md:51]: Same as above.

**Pull request merged**